File: Sondage.java
import java.util.ArrayList;
public class Sondage {
    public int idSondage;
    public String questionSondage;
    public ArrayList<Reponse> reponses;

    public Sondage (int idSondage, String questionSondage, ArrayList<Reponse> reponses) {
        this.idSondage = idSondage;
        this.questionSondage = questionSondage;
        this.reponses = reponses;
    }
}

File: Web/model/Sondage.php
<?php

public class Sondage {
    public $idSondage;
    public $questionSondage;
    public $idReponses;
    public $idEnseignant;

    public Sondage ($idSondage, $questionSondage, $reponses, $idEnseignant) {
        $this->idSondage = $idSondage;
        $this->questionSondage = $questionSondage;
        $this->reponses = $reponses;
        $this->idEnseignant = $idEnseignant;
    }
}
?>
